test/verifiedTests/analysis/testMultiSpeciesModelling/testSimulatePairwiseInteractions.m

```diff
@@ -0,0 +1,119 @@
+% The COBRAToolbox: testSimulatePairwiseInteractions.m
+%
+% Purpose:
+%     - ensure that pairwise and single growth rates are predicted and
+%     correctly interpreted in the function simulatePairwiseInteractions.
+%
+%       tests the following: if the interaction was interpreted beneficial for
+%       one species,the growth rate for the species has to be at least 10% higher
+%       in co-growth compared with single growth. If the interaction was
+%       interpreted detrimental for one species,the growth rate for the species
+%       has to be at least 10% lower in co-growth than in single growth. If the
+%       interaction was considered neutral for one species, the growth rate has
+%       to differ by less than 10% in co-growth compared with single growth.
+%
+% Author:
+%     - Almut Heinken - March 2017
+%     - CI integration - Laurent Heirendt - March 2017
+%     - renamed script from testSimulationPairwiseInteractions to
+%       testSimulatePairwiseInteractions and adapted to the new module
+%       Microbiome_Modeling_Toolbox - Almut Heinken - 02/2018
+
+currentDir = pwd;
+
+fileDir = fileparts(which('testSimulatePairwiseInteractions'));
+cd(fileDir);
+
+% if the pairedModelInfo file does not exist yet, build the models first
+if ~exist('pairedModelInfo', 'var')
+    modelList={
+        'Abiotrophia_defectiva_ATCC_49176'
+        'Acidaminococcus_fermentans_DSM_20731'
+        'Acidaminococcus_intestini_RyC_MR95'
+        'Acidaminococcus_sp_D21'
+        'Acinetobacter_calcoaceticus_PHEA_2'
+        };
+    for i=1:length(modelList)
+        load(modelList{i});
```

Comment:
as above, please use 
`model = getDistributedModel([modelList{i} '.mat']);`

Reply:
Hi @tpfau ,
I will address your concerns today.
I am not sure how to handle the issue with BIGG models. Can someone comment why these models use _e instead of [e]? Changing that by default might cause other issues.

Reply:
BiGG uses a SBML-ID compliant definition of IDs: 
https://github.com/SBRG/bigg_models/wiki/BiGG-Models-ID-Specification-and-Guidelines

Since SBML IDs are restricted in the possible characters used (and [] are not allowed), they decided to specify _cc_TT as specifications for compartment (two lower case letters) and tissue (two capital letters) at the end of any identifier.
We currently only support a compartment ID (commonly `\[[a-z]\]$`, but many functions also allow `\[[^\[]+\]$`), and we don't yet have a definition on how multiple Tissues/Organisms should be indicated. 

For now, Please don't worry about this part. There is an internal discussion started about how we should handle these differences.